src/shared/bus-unit-util.c

```diff
@@ -764,6 +764,7 @@ static const struct {
         const char *result, *explanation;
 } explanations [] = {
         { "resources",   "of unavailable resources or another system error" },
+        { "protocol",    "the service did not take the steps required by its configuration" },
```

Comment:
hmm "configuration" might be misleading. most people would probably look into the daemon's own configuratoin when debugging this when they read this. Maybe simply add in a "unit" there, so that it becomes "… required by its unit configuration"...